Students survey

IngelaL opened the pull request:
Link to my deployed project: https://survey-students.netlify.app/

My reflections on how this week's project turned out:
I think it turned out ok! I kept it to the blue level and I am happy about it.

Things I'd like to have clarified or explained in more detail:
No, not really.

karanmann commented on code/src/App.js:
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
+import { Form } from './Form'
```
We should try and have a line space between Components and library imports

karanmann commented on code/src/App.js:
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
+import { Form } from './Form'
+
```
We could also maybe delete these line spaces :).

karanmann commented on code/src/Form.js:
```diff
@@ -0,0 +1,52 @@
+import { ThirdQuestion } from './components/ThirdQuestion'
+import { SubmitButton } from './components/SubmitButton'
+import { Summary } from './components/Summary'
+import { useState } from 'react'
```
we could add use state to the react import itseld i.e. import React, {useState} from 'react'

karanmann commented on code/src/Form.js:
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Header } from 'components/Header'
```
Same here the line space between Library and components.

karanmann commented on code/src/Form.js:
```diff
@@ -0,0 +1,52 @@
+import { SubmitButton } from './components/SubmitButton'
+import { Summary } from './components/Summary'
+import { useState } from 'react'
+
+
+
```
Dont need all the line spaces :)

karanmann commented on code/src/Form.js:
```diff
@@ -0,0 +1,52 @@
+    const [location, setLocation] = useState('');
+    const [text, setText] = useState('');
+    const [submit, setSubmit] = useState(false);
+
```
same here.

karanmann commented on code/src/components/SubmitButton.js:
```diff
@@ -0,0 +1,9 @@
+
+export const SubmitButton = () => {
+    return (
+        <div className='submit-button'>
```
we could add a tab index on the submit button as it does not navigate there using just tab key at the moment.

karanmann commented on code/src/components/Summary.js:
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
```
Same here

karanmann commented on code/src/components/ThirdQuestion.js:
```diff
@@ -0,0 +1,23 @@
+        <div className='third-question'>
+            <label><h2 className='text-header' tabIndex='0'>What was the main reason you wanted to start at the Technigo bootcamp and become a frontend developer?</h2></label>
+                <textarea
+                    aria-label='text area'
```
Its great that you used the text area and limiting the number of characters. It really looks very good on the site.

karanmann reviewed (changes requested):
Hi Ingela, This was such a good and clean code, I infact learned a few things out of it myself. The only thing I would look into  is the spacing and chaining of command like the one for import of react and use state. You have achieved the blue level as soon as you have added the tab index to the Submit button. It was a beautiful site :). I have requested changes to the above and then it is ready to be aproved.

IngelaL commented:
@karanmann Thanks for your review Karan! It is so nice that we can learn from each other. I have now cleaned up the code and added the tabindex on the submit button, and pushed the changes. 👍
